modules/monitoring/manifests/checks/mirror.pp

```diff
@@ -8,13 +8,57 @@
 #   Can be used to disable checks/alerts for a given environment.
 #   Default: false
 #
+# [*gcp_mirror_sync_project_id*]
+#   Sets the GCP project id to use for the transfer job.
+#   Default: ''
```

Comment:
This contradicts the actual code. You've got these defaulting to `undef` (not empty string), which makes sense because you presumably want them to be optional so that they can be omitted when `enabled = false` but still raise an error if they're not set when they're needed.
```suggestion
#   Required if enabled=true.
```

(similarly for the others)

Reply:
Cheers, updated.